Utilities/types.h

```diff
@@ -541,7 +541,7 @@ constexpr u32 operator""_u32(const char* s, std::size_t length)
 // Convert 8-byte string to u64 value like reinterpret_cast does
 constexpr u64 operator""_u64(const char* s, std::size_t length)
 {
-	return length != 8 ? throw s :
+	return
```

Comment:
Consider for each character to check existence, for example character 8 can be read as: `to_u8(length >= 8 ? s[7] : '\0')`

Reply:
That looks redundant unless there is some use case for shorter strings. It can just become "consteval" function in C++20, and explicit length check will become redundant.